### ISSUE TITLE
No such file or directory: 'sshuttle-telepresence'

### ISSUE THREAD

**Issue by utrack** [telepresenceio/telepresence#688]
### What were you trying to do?
Run `telepresence`.

### What did you expect to happen?
TP should run normally and forward connections

### What happened instead?
Command had failed after
```
sudo: no tty present and no askpass program specified
Guessing that Services IP range is 10.96.0.0/12. Services started after this point will be inaccessible if are outside this range; restart telepresence if you can't access a new Service.
```

### Automatically included information

Command line: `['/opt/telepresence/telepresence-venv/bin/telepresence']`
Version: `0.88`
Python version: `3.6.5 (default, May 11 2018, 04:00:52) 
[GCC 8.1.0]`
kubectl version: `Client Version: v1.10.3 // Server Version: v1.10.4`
oc version: `(error: [Errno 2] No such file or directory: 'oc': 'oc')`
OS: `Linux galley 4.16.12-1-ARCH #1 SMP PREEMPT Fri May 25 23:30:31 UTC 2018 x86_64 GNU/Linux`
Traceback:

```
Traceback (most recent call last):
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/cli.py", line 87, in call_f
    return f(*args, **kwargs)
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/main.py", line 110, in main
    mount_dir
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/local.py", line 167, in run_local_command
    connect_sshuttle(runner, remote_info, args, subprocesses, env, ssh)
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/vpn.py", line 281, in connect_sshuttle
    runner, args, remote_info, env["KUBERNETES_SERVICE_HOST"]
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/runner.py", line 208, in popen
    track, out_cb, err_cb, args, done=done, **kwargs
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/runner.py", line 138, in launch_command
    process = launch_command(args, out_cb, err_cb, **kwargs)
  File "/opt/telepresence/telepresence-venv/lib/python3.6/site-packages/telepresence/runner.py", line 274, in launch_command
    process = Popen(args, **kwargs)
  File "/usr/lib64/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sshuttle-telepresence': 'sshuttle-telepresence'

```

Logs:

```
rictHostKeyChecking=no -o UserKnownHostsFile=/dev/null telepresence@localhost:/ /tmp/tmphix9m0ve
  21.2  54 | Handling connection for 33251
  21.9 TEL | END SPAN mount.py:36(mount_remote_volumes)    0.8s
  21.9 TEL | BEGIN SPAN vpn.py:258(connect_sshuttle)
  21.9 TEL | BEGIN SPAN vpn.py:84(get_proxy_cidrs)
  21.9 TEL | [61] Capturing: kubectl get nodes -o json
  22.1 TEL | [62] Capturing: kubectl get services -o json
  22.3 TEL | END SPAN vpn.py:84(get_proxy_cidrs)    0.4s
  22.3 TEL | [63] Launching: sshuttle-telepresence -v --dns --method nat -e 'ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -F /dev/null' --to-ns 127.0.0.1:9053 -r telepresence@localhost:33251 10.20.25.192/26 10.20.26.128/26 10.20.26.64/26 10.96.0.0/12 10.20.25.0/26 10.20.24.128/26 10.20.26.0/26 10.20.24.64/26 10.20.24.0/26 10.20.26.192/26 10.20.24.192/26 10.20.25.128/26 10.20.25.64/26 10.20.27.0/26
  22.3 TEL | [63] [Errno 2] No such file or directory: 'sshuttle-telepresence': 'sshuttle-telepresence'

```

**Comment by ark3** [comment]
Sorry about the crash! Telepresence should notice that its helper is unavailable and report that cleanly.

Are you using Arch Linux? How did you install Telepresence? 

Using the [install from source](https://www.telepresence.io/reference/install#install-from-source) method should include `sshuttle-telepresence`, thereby avoiding this problem.

**Comment by utrack** [comment]
Thank you! Yeah, I guess bad package is the root cause. 

I've installed `telepresence` AUR package on Arch.

**Comment by ark3** [comment]
See also the comments in #135, which includes a pointer to another AUR that is more up-to-date.

**Comment by ark3** [comment]
I believe this was fixed in 0ecd9e6.
